cmd/crc/cmd/podman_env.go

```diff
@@ -0,0 +1,46 @@
+package cmd
+
+import (
+	"github.com/code-ready/crc/pkg/crc/constants"
+	"github.com/code-ready/crc/pkg/crc/errors"
+	"github.com/code-ready/crc/pkg/crc/machine"
+	"github.com/code-ready/crc/pkg/crc/output"
+	"github.com/code-ready/crc/pkg/os/shell"
+	"github.com/spf13/cobra"
+)
+
+var podmanEnvCmd = &cobra.Command{
+	Use:   "podman-env",
+	Short: "Setup podman environment",
+	Long:  `Setup environment for 'podman' binary to access podman on CRC VM`,
+	Run: func(cmd *cobra.Command, args []string) {
+		userShell, err := shell.GetShell(forceShell)
+		if err != nil {
+			errors.ExitWithMessage(1, "Error running the podman-env command: %s", err.Error())
+		}
+
+		ipConfig := machine.IpConfig{
+			Name:  constants.DefaultName,
+			Debug: isDebugLog(),
+		}
+
+		exitIfMachineMissing(ipConfig.Name)
+
+		result, err := machine.Ip(ipConfig)
+		if err != nil {
+			errors.Exit(1)
+		}
+
+		output.Outln(shell.GetPathEnvString(userShell, constants.CrcBinDir))
+		output.Outln(shell.GetEnvString(userShell, "PODMAN_USER", constants.DefaultSSHUser))
+		output.Outln(shell.GetEnvString(userShell, "PODMAN_HOST", result.IP))
+		output.Outln(shell.GetEnvString(userShell, "PODMAN_IDENTITY_FILE", constants.GetPublicKeyPath()))
+		output.Outln(shell.GetEnvString(userShell, "PODMAN_IGNORE_HOSTS", "1"))
```

Comment:
I'm not sure what this one is doing? I could not find much doc around it

Reply:
i don't know either actually, I just added this cause the original config @praveenkumar tried had this, so i just kept this around. Maybe Praveen knows?

Reply:
@cfergeau this is part of https://github.com/containers/libpod/blob/7af4074c6f96d0fabd2ab5dd6c5773053f80aa08/cmd/podman/main_remote.go one and not in the documents, may be better to have it as part of commit logs also. I am not sure if podman team have any plan to include those env in the documents.

Reply:
would it be possible to use a template instead?

Reply:
I removed the template stuff from `oc-env` as in my opinion it made the code harder to read for no real gain.

Reply:
when considering all platform,s, it would be easy to put all templates specifc to a platform in a platform typed source code file. now it is all contained inside the actual code and harder to maintain too... specially as the environment/shell is harder to determine to be correct in the current way.

Reply:
... even if the template is not consider, the name `GetEnvString` does not seem correct when you realize what it does.

Reply:
Instead of this we could also create the config file for podman-remote. https://github.com/containers/libpod/blob/master/docs/source/markdown/podman-remote.conf.5.md

Reply:
if you check the issue itself we decided against this, as that would mess up exisitng and future connections. now they are scoped to the terminal session.